src/Makefile

```diff
@@ -95,16 +95,16 @@ all: redisgraph.so
 
 $(LIBTRIEMAP):
 	$(MAKE) -C util/triemap
-.PHONY: $(LIBTRIEMAP)
 
 BLAS_CFLAGS = -fPIC -I $(abs_path)/util -include rmalloc.h -DREDIS_MODULE_TARGET -DGB_MALLOC=rm_malloc -DGB_CALLOC=rm_calloc -DGB_REALLOC=rm_realloc -DGB_FREE=rm_free -DGBCOMPACT
 
 # Build GraphBLAS only if library does not already exists.
 $(GRAPHBLAS):
 ifeq (,$(wildcard $(GRAPHBLAS)))
 	$(MAKE) CMAKE_OPTIONS="-DUSER_POSIX=1 -DCMAKE_CXX_FLAGS=-std=c++11 -DCMAKE_C_FLAGS='$(BLAS_CFLAGS)'" static_only -C ../deps/GraphBLAS
+else
+
```

Comment:
What's the purpose of an empty else block?

Reply:
One should keep an empty line after the $(GRAPHBLAS): line to indicate an empty rule.